src/components/Header/Header.tsx

```diff
@@ -1,9 +1,16 @@
-import Navbar from "./Navbar/Navbar";
+// import Navbar from "./Navbar/Navbar";
 
 export default function Header() {
   return (
     <header>
-      <Navbar />
+      <div className="w-full flex justify-center mt-5 md:block">
+        <a
+          href="/"
+          className="md:mt-7 md:ml-7 block transform text-white text-5xl font-kalnia z-20 hover:text-gray-300"
+        >
+          Wedgy.
+        </a>
+      </div>

```

Comment:
```suggestion
      <header className="w-full flex justify-center mt-5 md:block">
        <a
          href="/"
          className="md:mt-7 md:ml-7 block transform text-white text-5xl font-kalnia z-20 hover:text-gray-300"
        >
         <h1> Wedgy. <h1>
        </a>
      </header>

```
Suggest removing unnecessary div (styling could go directly into header element) and making the h1 element for the page here